server/react-admin/src/videoTestimonials/VideoTestimonialForm.js

```diff
@@ -75,16 +75,12 @@ function VideoTestimonial() {
               />
             </Grid>
             <Grid item lg={4} md={6} xs={12}>
-              <input
-                type="checkbox"
-                name="privacyStatement"
-                value={privacyStatement}
-                onClick={() => {
-                  setPrivacyStatement(!privacyStatement)
-                }}
+              <CheckboxInput
+                name="privacy_stmt_ack"
+                control={control}
+                label={'Privacy Statement Acknowledgement'}
                 required
```

Comment:
@markstur Is the intent to have this always checked? If it is `required` then they won't be able to save it without checking the box.

Reply:
If it isn't, in place of an error we could popup a modal telling them what acknowledging or not acknowledging the privacy statement means for them.